packages/front-end/components/Radix/Dropdown.tsx

```diff
@@ -0,0 +1,78 @@
+// radix dropdown wrapper
+import { DropdownMenu as RadixDropdownMenu } from "@radix-ui/themes";
+// import colors from radix-ui themes
+import type { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+type AllowedChildren = React.ReactNode;
+
+type Props = {
+  trigger: React.ReactNode;
+  disabled?: boolean;
+  errorLevel?: "error" | "warning";
+  description?: string;
```

Comment:
Are these radix dropdown props? I wasn't able to find them in the API reference here: https://www.radix-ui.com/themes/docs/components/dropdown-menu#root